src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaCollection.js

```diff
@@ -108,25 +118,24 @@ class MediaCollection extends React.Component<Props> {
                 open={uploadOverlayOpen}
             >
                 <CollectionSection
-                    addable={addable}
-                    deletable={deletable}
-                    editable={editable}
+                    addable={!locked && addPermission}
+                    deletable={!locked && deletePermission}
+                    editable={!locked && editPermission}
                     listStore={collectionListStore}
                     locale={locale}
                     onCollectionNavigate={this.handleCollectionNavigate}
                     overlayType={overlayType}
                     resourceStore={collectionStore.resourceStore}
-                    securable={securable}
+                    securable={!locked && securityPermission}
                 />
                 <Divider />
-                <div>
-                    <MediaSection
-                        adapters={mediaListAdapters}
-                        listStore={mediaListStore}
-                        mediaListRef={mediaListRef}
-                        onMediaClick={this.handleMediaClick}
-                    />
-                </div>
+                <List
```

Comment:
decided to directly render a `List` instead of the `MediaSection` because the `MediaSection` does not really do anything and I would need to add a `actions` prop to the `MediaSection`